cypress/integration/add_item.js

```diff
@@ -2,24 +2,37 @@ describe("Add Item To List", function() {
   beforeEach(function() {
     window.localStorage.setItem("uniqueToken", "token1234");
   });
-
```

Comment:
Is there a reason you're deleting all of the empty lines between tests?

Reply:
@mindyzwan I did not reaIize we had deleted empty lines, I can add them back it :)